db/migrate/20160405143543_create_git_references.rb

```diff
@@ -0,0 +1,14 @@
+class CreateGitReferences < ActiveRecord::Migration[5.0]
+  def change
+    create_table :git_references do |t|
+      t.string    :name
+      t.string    :commit_sha
+      t.timestamp :commit_time
+      t.text      :commit_message
+      t.string    :type
+      t.bigint    :git_repository_id
+
+      t.timestamps
```

Comment:
Seems legit... I vaguely recall seeing that and this confirms it:  834dab6366621351bdc658ac34b3eb89ab68b9c0, see also:  http://juanitofatas.com/2015/01/28/rails-5-timestamps/